itou/siae_evaluations/models.py

```diff
@@ -0,0 +1,145 @@
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db import models
+from django.urls import reverse
+from django.utils import timezone
+
+from itou.institutions.models import Institution
+from itou.siae_evaluations import enums as evaluation_enums
+from itou.utils.emails import get_email_message
+
+
+def validate_institution(institution_id):
+    try:
+        institution = Institution.objects.get(pk=institution_id)
+    except Institution.DoesNotExist:
+        raise ValidationError("L'institution sélectionnée n'existe pas.")
+
+    if institution.kind != Institution.Kind.DDETS:
+        raise ValidationError(f"Sélectionnez une institution de type {Institution.Kind.DDETS}")
+
+
+def email_campaign_is_setup(emails, evaluated_period_start_at, evaluated_period_end_at, ratio_selection_end_at):
+    to = [settings.DEFAULT_FROM_EMAIL]
+    bcc = emails
+    context = {
+        "evaluated_period_start_at": evaluated_period_start_at,
+        "evaluated_period_end_at": evaluated_period_end_at,
+        "ratio_selection_end_at": ratio_selection_end_at,
+        "dashboard_url": f"{settings.ITOU_PROTOCOL}://{settings.ITOU_FQDN}{reverse('dashboard:index')}",
+    }
+    subject = "siae_evaluations/email/campaign_is_setup_subject.txt"
+    body = "siae_evaluations/email/campaign_is_setup_body.txt"
+    return get_email_message(to, context, subject, body, bcc=bcc)
```

Comment:
je ne sais pas si tu as besoin de toutes ces variables intermédiaires, tu pourrais aussi bien appeler directement `get_email_message` dans `create_campaign en passant les arguments directement sans créer `to`, `bcc`, etc.

Reply:
mais bon c'est tres mineur comme commentaire